.github/workflows/vertexai.yml

```diff
@@ -43,13 +43,14 @@ jobs:
   spm-integration:
     strategy:
       matrix:
-        target: [macOS]
+        target: [iOS]
         os: [macos-14]
         include:
           - os: macos-14
             xcode: Xcode_15.2
     runs-on: ${{ matrix.os }}
     env:
+      TEST_RUNNER_VertexAIRunIntegrationTests: 1
```

Comment:
TIL you need to prefix with `TEST_RUNNER_` for it to make it into the environment variables in the XCTest:

> `xcodebuild` now supports passing certain environment variables to test runner processes. In the environment where `xcodebuild` is invoked, prefix any variable with `TEST_RUNNER_` to pass that variable (with the prefix stripped) to XCTest test runner processes. For example, running `env TEST_RUNNER_Foo=Bar xcodebuild test ...` causes the environment variable `Foo=Bar` to be set in the test runner’s environment. (74104870)

-- https://developer.apple.com/documentation/xcode-release-notes/xcode-13-release-notes#Testing

Although I am using Xcode 15.2, not 15.3, I stumbled across this from https://forums.developer.apple.com/forums/thread/749185.

Reply:
Good find! TIL, too!